themes/custom/campaign_base/Gruntfile.js

```diff
@@ -74,7 +74,12 @@ module.exports = function (grunt) {
           separator: ';',
       },
       dist: {
```

Comment:
@AndyEPhipps I think we should keep external libraries separated from our theme `campaign_base.js`. It's ok to have our patternlab components js files in campaign_base.js because they are part of our theme.

For lightcase we have added it via `campaign_base.libraries.yml`.
```global-js:
  js:
    node_modules/lightcase/src/js/lightcase.js: {}
    libraries/modernizr/modernizr-custom.js: {}
    js/campaign_base.min.js: {}```